test/inductor/test_b2b_gemm.py

```diff
@@ -1,40 +1,89 @@
+    def test_b2b_gemm_applicable(self):
+        """applicable sizes"""
 
         def f(m1: torch.Tensor, m2: torch.Tensor, m3: torch.Tensor) -> torch.Tensor:
-            return torch.matmul(torch.matmul(m1, m2), m3)
+            return torch.mm(torch.mm(m1, m2), m3)
 
         f_opt = torch.compile(f)
-        A = torch.randn((512, 64), device="cuda", dtype=torch.float16)
-        B = torch.randn((64, 256), device="cuda", dtype=torch.float16)
-        C = torch.randn((256, 64), device="cuda", dtype=torch.float16)
+        A = torch.randn((1024, 10), device="cuda", dtype=torch.float16)
+        B = torch.randn((10, 1024), device="cuda", dtype=torch.float16)
+        C = torch.randn((1024, 10), device="cuda", dtype=torch.float16)
         res, (code,) = run_and_get_code(f_opt, A, B, C)
         self.assertTrue(torch.allclose(f(A, B, C), res, atol=0.2, rtol=0.01))
         self.assertTrue("B2B_GEMM_TRITON_ENTRANCE" in code)
 
+    @torch._dynamo.config.patch(cache_size_limit=32)
     @torch._inductor.config.patch(b2b_gemm_pass=True)
-    def test_b2b_gemm_not_divisible(self):
-        """sizes are not divisible by block sizes"""
+    def test_b2b_gemm_not_applicable(self):
+        """non-applicable sizes"""
 
         def f(m1: torch.Tensor, m2: torch.Tensor, m3: torch.Tensor) -> torch.Tensor:
-            return torch.matmul(torch.matmul(m1, m2), m3)
+            return torch.mm(torch.mm(m1, m2), m3)
 
         f_opt = torch.compile(f)
         A = torch.randn((500, 100), device="cuda", dtype=torch.float16)
         B = torch.randn((100, 200), device="cuda", dtype=torch.float16)
         C = torch.randn((200, 300), device="cuda", dtype=torch.float16)
         res, (code,) = run_and_get_code(f_opt, A, B, C)
         self.assertTrue(torch.allclose(f(A, B, C), res, atol=0.2, rtol=0.01))
-        self.assertTrue("B2B_GEMM_TRITON_ENTRANCE" in code)
+        self.assertTrue("B2B_GEMM_TRITON_ENTRANCE" not in code)
+
+    @torch._dynamo.config.patch(cache_size_limit=32)
+    def test_b2b_gemm_performance(self):
+        """compare torch.compile(f, b2b_gemm = off) with torch.compile(f, b2b_gemm = on)"""
+
+        def run_with_b2b_gemm_off(m1: torch.Tensor, m2: torch.Tensor, m3: torch.Tensor) -> float:
+
+            def f(m1: torch.Tensor, m2: torch.Tensor, m3: torch.Tensor) -> torch.Tensor:
+                return torch.mm(torch.mm(m1, m2), m3)
+
+            f_opt = torch.compile(f, dynamic=False)
+            return do_bench(f_opt, (m1, m2, m3), {}, warmup = 200, rep = 2000)
+
+        @torch._inductor.config.patch(b2b_gemm_pass=True)
+        def run_with_b2b_gemm_on(m1: torch.Tensor, m2: torch.Tensor, m3: torch.Tensor) -> float:
+
+            def f(m1: torch.Tensor, m2: torch.Tensor, m3: torch.Tensor) -> torch.Tensor:
+                return torch.mm(torch.mm(m1, m2), m3)
+
+            f_opt = torch.compile(f, dynamic=False)
+            return do_bench(f_opt, (m1, m2, m3), {}, warmup = 200, rep = 2000)
+
+        speedups = []
+        print()
```

Comment:
Assuming we keep this test, I'd vote to remove `print`s. If the console output is useful, let's at least use a `log.debug`